### PR TITLE
MDP-211: Refactor footer

### PR DESCRIPTION
**Pull request by PavolKiss**

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -7,43 +7,45 @@ import { Contacts } from "./Contacts";
 import { Location } from "./Location";
 import Instagram from "../../static/svg/instagram.svg";
 import Facebook from "../../static/svg/facebook-square.svg";
+import { Link } from "@reach/router";
```
Comment (pggl) [inline src/components/Footer/Footer.tsx]: we don't need router for this, this is external link

File: src/components/Footer/Info.tsx [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-      OZ CHAT - Centrum pre kreatívnu liečbu Arteterapiou
-    </div>
-    <div className="text-10 font-lato font-normal color-black">
-      <div>IČO: 42407681</div>
-      <div>Číslo účtu: VÚB SK13 0200 0000 0035 0237 4057</div>
-    </div>
+interface IProps {
+  title: string;
+  ico: number;
+  bankNumber: string;
```
Comment (pggl) [inline src/components/Footer/Info.tsx]: we don't need this props, this should be hardcoded we don't want to change this or reuse in different place

File: src/components/Footer/Contacts.tsx [hunk 3]
```diff
@@ -1,28 +1,29 @@
...
 import Email from "../../static/svg/email.svg";
 import { Icon } from "../Icon";
+import { Link } from "@reach/router";
 
-export const Contacts = () => (
-  <div className="ml-24 mt-8 lg:mt-0 sm:mt-0 mr-16">
-    <div className="pb-4 flex justify-start items-center">
+interface IProps {
+  phoneNumber: number;
+  email: string;
```
Comment (pggl) [inline src/components/Footer/Contacts.tsx]: we don't need this props, this should be hardcoded we don't want to change this or reuse in different place

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/Footer/Contacts.tsx [hunk 4]
```diff
@@ -2,27 +2,23 @@ import React from "react";
...
       </div>
-      <div className="font-lato text-10">
-        <a
-          href="mailto:chatkosice@gmail.com"
-          className="no-underline text-black"
-        >
-          chatkosice@gmail.com
-        </a>
-      </div>
+      <Link to="mailto:chatkosice@gmail.com" className="font-lato text-10">
```
Comment (pggl) [inline src/components/Footer/Contacts.tsx]: this should be <a> tag, Link is for internal routing

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/Footer/Info.tsx [hunk 5]
```diff
@@ -1,13 +1,9 @@
...
-  <div className="pt-20 pl-24 h-40">
-    <div className="text-10 font-lato font-black">
-      OZ CHAT - Centrum pre kreatívnu liečbu Arteterapiou
-    </div>
-    <div className="text-10 font-lato font-normal color-black">
-      <div>IČO: 42407681</div>
-      <div>Číslo účtu: VÚB SK13 0200 0000 0035 0237 4057</div>
-    </div>
+  <div className="mt-20 ml-24 h-40 text-10 font-lato">
+    <div>OZ CHAT - Centrum pre kreatívnu liečbu Arteterapiou</div>
```
Comment (slavomircesla) [inline src/components/Footer/Info.tsx]: Bold text

File: src/components/Footer/Contacts.tsx [hunk 6]
```diff
@@ -2,27 +2,23 @@ import React from "react";
 import Phone from "../../static/svg/icon-phone.svg";
 import Email from "../../static/svg/email.svg";
 import { Icon } from "../Icon";
+import { Link } from "@reach/router";
```
Comment (slavomircesla) [inline src/components/Footer/Contacts.tsx]: Not needed I suppose.

File: src/components/Footer/Info.tsx [hunk 7]
```diff
@@ -1,13 +1,11 @@
 import React from "react";
 
 export const Info = () => (
-  <div className="pt-20 pl-24 h-40">
-    <div className="text-10 font-lato font-black">
+  <div className="mt-20 ml-24 h-40 text-10 font-lato">
```
Comment (slavomircesla) [inline src/components/Footer/Info.tsx]: text-12 in mobile zeplin?

**Review by pggl — APPROVED** [review]

**Pull request merged**